templates/Deepcell_Postprocessing.ipynb

```diff
@@ -130,11 +131,12 @@
    "source": [
     "# get input data for overlay\n",
     "input_data_xr = data_utils.load_imgs_from_multitiff(deepcell_input_dir, multitiff_files=points_input)\n",
+    "overlay_channels = [input_data_xr.channels.values[:2]]\n",
```

Comment:
Any reason we're stripping this list to include just the first two elements? Feels like a somewhat arbitrary decision. @ackagel @ngreenwald feel free to let me know if there's a good reason for this.

Reply:
The deepcell input data should only have two channels (summed nuclear and summed membrane).  Trimming shouldn't affect the results, but it does prevent weird overlays if we end up adding more channels to deepcell inputs.  imo it could stay or go, no real preference here.

Reply:
I think I addressed this earlier; we should use channel names so it's clear what's happening here instead of indexing